filebeat/input/filestream/internal/input-logfile/input.go

```diff
@@ -34,6 +34,7 @@ type managedInput struct {
 	prospector   Prospector
 	harvester    Harvester
 	cleanTimeout time.Duration
+	readerLimit  uint32
```

Comment:
If I understand the PR correctly, the limit is indeed local per configured Input. In case of auto-discovery for too many containers this won't help much. We have plans to also add a global limit at the level of the input manager as well?

Reply:
Yes, I would like to let users limit the number of harvesters both on the input level and globally. If someone already uses `harvester_limit` and satisfied with it, I do not want to take it away from them. The global limit is supposed to come in a follow-up PR, but I can add it here if it's better.